keras/src/quantizers/gptqutils.py

```diff
@@ -0,0 +1,419 @@
+import io
+import random
+import tarfile
+
+import numpy as np
+import requests
+from absl import logging
+from datasets import load_dataset
```

Comment:
requests for downloading data makes the code fragile

Reply:
Removed it